test/codegen/array.c

```diff
@@ -22,5 +22,6 @@ int main() {
   __builtin_print(e[1]);
   __builtin_print(e[2]);
 
+  int f[2];
```

Comment:
Add some comment stating that the local array should not be 0-initialized, and the generated IR has to be checked manually.